src/Person.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by JacobP on 2/7/17.
+ */
+public class Person {
+    int age;
+    double height;
+    double weight;
+    String eyeColor;
+    boolean isMarried;
+
```

Comment:
the person class isn't in the same package as the other classes, but this is a minor detail. Really nice job overall.